From dd9050361d36380020459820940321d476d02926 Mon Sep 17 00:00:00 2001
From: Mark Washeim <blueprint@poetaster.de>
Date: Wed, 26 Apr 2023 09:44:55 +0200
Subject: [PATCH] Patch: remove optional KF5DocTools

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3d4302f80..7154e6b0a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,6 @@ find_package(KF5Config ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5CoreAddons ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5I18n ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5Service ${KF_DEP_VERSION} REQUIRED)
-find_package(KF5DocTools ${KF_DEP_VERSION})
 find_package(KF5Solid ${KF_DEP_VERSION} REQUIRED) # for kio_trash
 if (NOT ANDROID)
     find_package(KF5Crash ${KF_DEP_VERSION} REQUIRED)
-- 
2.34.1

